src/Parser/Rogue/Assassination/Modules/Legendaries/DuskwalkersFootpads.js

```diff
@@ -27,7 +29,8 @@ class DuskwalkersFootpads extends Analyzer {
   }
 
   on_byPlayer_spendresource(event) {
-    if (event.resourceChangeType !== RESOURCE_TYPES.ENERGY.id) {
+    if (event.resourceChangeType !== RESOURCE_TYPES.ENERGY.id
+        || event.ability === SPELLS.FEINT.id) {
```

Comment:
matching the spellId `event.ability.guid === SPELLS.FEINT.id`, as event.ability is an object containing a few props about the ability.  Sorry there are so many things to work out here 😐 ...

Reply:
This is my first time contributing to an actual application; I'm new to this programming stuff. I expected a lot of kinks. Thanks for being so patient with me!

I'll get that fixed right away, thanks for the tip!